homeassistant/components/sensor/rmvtransport.py

```diff
@@ -146,9 +145,12 @@ def icon(self):
         return self._icon
 
     @property
-    def unit_of_measurement(self):
-        """Return the unit this state is expressed in."""
-        return "min"
+    def device_class(self):
+        """Return the device class of the sensor."""
+        return DEVICE_CLASS_TIMESTAMP
```

Comment:
Do not do that. You should first implement it as an existing device class, wait until #14983 merged, then open anther PR to change your sensor's device class. Otherwise I have to put your PR on hold until the dependency satisfied.  

I don't know when #14983 will merge.

Reply:
Fair enough.